docs/make.jl

```diff
@@ -2,6 +2,9 @@ using Documenter, AnalyzeRegistry
 
 DocMeta.setdocmeta!(AnalyzeRegistry, :DocTestSetup, :(using AnalyzeRegistry); recursive=true)
 
+previous_GKSwstype = get(ENV, "GKSwstype", "")
+ENV["GKSwstype"] = "100"
```

Comment:
You can use [`withenv`](https://docs.julialang.org/en/v1/base/base/#Base.withenv) around `makedocs` and `deploydocs`